packages/contract/src/util.ts

```diff
@@ -64,11 +63,9 @@ export const getContractId = (
  * Ensures that a string is hexlified.
  *
  * @param value - The value to be hexlified.
- * @param options - Options for hexlify.
- * @returns The input value hexlified.
+ * @returns The input value hexlified with prefix.
  */
-export const includeHexPrefix = (value: string, options?: DataOptions) =>
-  hexlify(value, {
-    ...options,
-    allowMissingPrefix: true,
-  });
+export const hexlifyWithPrefix = (value: string) => {
+  if (value.startsWith('0x')) return hexlify(value);
+  return hexlify(`0x${value}`);
+};
```

Comment:
`hexlify` v6 no longer takes any configuration options but the v5 hex prefix option is mirrored here for use in storage slots.